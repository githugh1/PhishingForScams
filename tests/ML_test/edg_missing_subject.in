{
  "email_body": "This is a test email.",
  "sender": "test@email.com"
}
