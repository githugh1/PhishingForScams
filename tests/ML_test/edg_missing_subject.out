{
  "error": "Missing email subject.",
  "message": "Please provide an email subject."
}
